templates/2answer.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
```

Comment:
Around your whole document (aside from the first line that declares document type), you need a '&lt;html&gt;&lt;/html&gt;' element.